### DISCUSSION TITLE
GaN thermal simulation

### DISCUSSION THREAD

**Discussion by tomhajjar** [ra3xdh/qucs_s#523] [Q&A]
Working on another ngspice "thermal" example. This time for for a GaN thermal simulation. The GaN device had a library model with thermal negating the need for Qucs-S to make a new component like for VDMOS thermal. 
https://ngspice.sourceforge.io/el-th-images/GaN-test.7z

I deleted the .spiceinit from my user folder so Qucs-S controls it.

![2024-01-28_112920](https://github.com/ra3xdh/qucs_s/assets/102838604/280ec454-820c-4d0f-8ae0-47815f68db4d)

[GaN_thermal_prj.zip](https://github.com/ra3xdh/qucs_s/files/14076800/GaN_thermal_prj.zip)

**Comment by ra3xdh** [comment]
I don't see any simulation issues in this example. I am transferring this issue to discussion.

**Comment by tomhajjar** [comment]
The example just points out the use of manufacturers "thermal" models when available. 

Unfortunately ngspice doesn't work for most of the MOSFET and IGBT thermal models I've tried. They do work in LTspice. The guys at ngspice haven't given any feedback yet on how to get these models to work. Hopefully I haven't overloaded them with requests...

https://sourceforge.net/p/ngspice/discussion/120973/thread/4721495ffa/
https://sourceforge.net/p/ngspice/discussion/120973/thread/b18b28cdab/

[thermal simulations.zip](https://github.com/ra3xdh/qucs_s/files/14143607/thermal.simulations.zip)
